mkdocs.yml

```
@@ -40,7 +41,12 @@ nav:
           - Reusable actions: actions-reusable.md
       - Jobs site: jobs-site.md
       - Level 4 server: level-4-server.md
-      - Safe Outputs and requesting release of files from the Level 4 server: releasing-files.md
+      - Requesting release of files from the Level 4 server:
+        - Overview: releasing-files-intro.md
+        - Applying statistical disclosure controls to your research outputs: sdc.md
+        - Requesting release of research outputs: requesting-file-release.md
+        - Review of outputs requested for release: output-checking.md
+        - File release: releasing-files.md
```

Some ideas to improve (I tried to shorten some, and use _clearer_ terms) and  the headings and subheadings. Ignore if you're happy with your titles

```suggestion
      - Releasing research outputs:
        - Overview: releasing-files-intro.md
        - Applying statistical disclosure controls: sdc.md
        - Requesting release of research outputs: requesting-file-release.md
        - Review process of output requests: output-checking.md
        - Release of approved outputs: releasing-files.md
```

---

In case you decide to make any changes to subheadings, probably also change the title of the page

---

I've left reference to Level 4 files in the main title - I think this is important, but have updated the subsection titles in line with your suggestions